CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Latest
 
 - fix: wildcard route matching
-- `func (*cookie.Cookie) MaxAge(int)`: use to modified max age field of the cookie
+- `func (\*cookie.Cookie) MaxAge(int)`: use to modified max age field of the cookie
```

Comment:
why has `\` here?

Reply:
to escape *

Reply:
But those code already in ``, I go back to master/CHANGELOG and didn't find the problem